scripts/pages/zoning/zoning.ts

```diff
@@ -532,24 +539,19 @@ class ZoneMenuHandler {
 
 		const index = this.panels.regionSelect.GetSelected()?.GetAttributeInt('value', -1);
 		const region = this.selectedZone.zone.regions[index];
+		this.selectedZone.region = region;
 
-		this.panels.pointsList.RemoveAndDeleteChildren();
-		if (region && region.points && region.points.length > 0) {
-			for (const [i, point] of region.points.entries()) {
-				this.addPointToList(i, point);
-			}
-		}
 		this.panels.regionBottom.text = (region?.bottom ?? 0).toFixed(2);
 		this.panels.regionHeight.text = (region?.height ?? 0).toFixed(2);
 		this.panels.regionSafeHeight.text = (region?.safeHeight ?? 0).toFixed(2);
 
-		const tpIndex = !region.teleDestTargetname
+		const tpIndex = (region.teleDestTargetname === undefined || region.teleDestTargetname === '')
```

Comment:
Empty string (and only the empty string) is falsy so original seems fine? Unless you're concerned about it being `null`, in which case I'd maybe rethink this code since combining `undefined` and `null` like this tends to lead to very confusing code.